modules/nf-core/atlas/splitmerge/meta.yml

```diff
@@ -7,58 +7,63 @@ keywords:
   - read group
 tools:
   - "atlas":
-      description: "ATLAS, a suite of methods to accurately genotype and estimate genetic diversity"
+      description: "ATLAS, a suite of methods to accurately genotype and estimate genetic
+        diversity"
       homepage: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
       documentation: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
       tool_dev_url: "https://bitbucket.org/wegmannlab/atlas"
       doi: "10.1101/105346"
-      licence: "['GPL v3']"
+      licence: ["GPL v3"]
+      identifier: biotools:atlas_db
 input:
-  - meta:
-      type: map
-      description: |
-        Groovy Map containing sample information
-        e.g. [ id:'test', single_end:false ]
-  - bam:
-      type: file
-      description: Single input BAM file.
-      pattern: "*.bam"
-  - bai:
-      type: file
-      description: The BAI file for the input BAM file
-      pattern: "*.bai"
-  - read_group_setting:
-      type: file
-      description: |
-        TXT file containing the split and merge settings for
-        each readgroup. Each line consist of one readgroup,
-        single/double identifier and the maximum cycle number
-        of the sequencer. e.g. "RG1 single 100"
-      pattern: "*.txt"
-  - blacklist:
-      type: file
-      description: |
-        blacklist.txt (optional), A txt file with blacklisted read names
-        that should be ignored and just written to file, each on a new line
-      pattern: "*.txt"
+  - - meta:
+        type: map
```

Comment:
I think in the comments we said that we'd remove this, right? But it seems to still be in the diff.

Reply:
Ah wait ignore me, it was a key called `qualifier`. Sorry.